systers_portal/community/apps.py

```diff
@@ -0,0 +1,9 @@
+from django.apps import AppConfig
+
+
+class CommunityConfig(AppConfig):
+    name = 'community'
+
+    def ready(self):
+        import community.signals
+        assert community.signals
```

Comment:
assert is added to remove `imported but unused` pep8 error. https://groups.google.com/forum/#!topic/comp.lang.python/nryFbibthpo